src/components/Dashboard/TodoList.vue

```diff
@@ -1326,7 +1328,7 @@ export default class TodoList extends Mixins(
           (item.correctedFilingType === FilingNames.CHANGE_OF_REGISTRATION) ||
           (item.correctedFilingType === FilingNames.REGISTRATION)
         ) {
-          // navigate to Edit UI to resume this correction
+          // navigate to Edit UI to resume this correction (depending on either CLIENT OR STAFF)
```

Comment:
This navigation does not depend on CLIENT or STAFF. What are you pointing out here?